app/views/events/edit.html.erb

```diff
@@ -1,6 +1,9 @@
-<h1>Editing Event</h1>
+<div class="container">
+  <h1>Editing Event</h1>
 
-<%= render 'form', event: @event %>
-
-<%= link_to 'Show', @event %> |
-<%= link_to 'Back', events_path %>
+  <%= render 'form', event: @event %>
+  
+  <br>
+  <%= link_to '<button type="button", class="btn btn-primary btn-sm">show</button>'.html_safe, @event%>
```

Comment:
If you want to have a link like a button you just need to add class for that. Like this: `link_to 'show', @event, class: 'btn btn-primary btn-sm'`

Reply:
Here you have a [doc](https://api.rubyonrails.org/v5.2.3/classes/ActionView/Helpers/UrlHelper.html) for that.